settings.php

```diff
@@ -10,10 +10,32 @@
  * Most sites can configure their database by entering the connection string
  * below. If using master/slave databases or multiple connections, see the
  * advanced database documentation at
- * https://api.backdropcms.org/database-configuration
+ * https://docs.backdropcms.org/database-configuration
+ *
+ * If your server uses a different port for the database from the default of
+ * '3306', you can add the 'port' attribute to the array.
```

Comment:
```suggestion
 * '3306', you can add the 'port' key to the array.
```
Lets also mention which of these keys are optional, and what they default to if omitted.

Reply:
Done - I think I changed it to that because of the Harry Potter association of port key

Reply:
I was a bit confused by this:

> Lets also mention which of these keys are optional, and what they default to if omitted.

Please can you elaborate?

Reply:
Discussion about optional keys moved to issue.